### PR TITLE
Remove several fs commands

### PR DESCRIPTION
**Pull request by yyongycy**
### What changes are proposed in this pull request?

Remove several fs commands:
getFacl
Persist
Pin
setFacl
setTTL
unpin
unsetTTL

### Why are the changes needed?

Remove several fs commands, those are no longer used or valid.

### Does this PR introduce any user facing changes?

NA

### CODE REVIEW

**Comment by yyongycy** [comment]
@huanghua78 @dbw9580  Please help review

**Review by huanghua78 — APPROVED** [review]
looks good

File: dora/shell/src/main/java/alluxio/cli/fs/command/SetTtlCommand.java [hunk 1]
```diff
@@ -1,114 +0,0 @@
...
-import java.io.IOException;
-import javax.annotation.concurrent.ThreadSafe;
-
-/**
- * Sets a new TTL value for the file at path both of the TTL value and the path are specified by
- * args.
- */
-@ThreadSafe
-@PublicApi
-public final class SetTtlCommand extends AbstractFileSystemCommand {
```
Comment (jiacheliu3) [inline dora/shell/src/main/java/alluxio/cli/fs/command/SetTtlCommand.java:37]: i think TTL still exists in Dora? Don't you want a command to manually override something? @huanghua78

**Review by jiacheliu3 — APPROVED** [review]
LGTM except one comment. Thanks!

File: dora/shell/src/main/java/alluxio/cli/fs/command/SetTtlCommand.java (hunk 1 above)
Comment (yyongycy) [reply dora/shell/src/main/java/alluxio/cli/fs/command/SetTtlCommand.java:37]: yesterday we had a discussion, most likely it is not valid in dora's env, and need to re-implement.

**Comment by yyongycy** [comment]
alluxio-bot, merge this please

**Pull request merged**